astropy/units/quantity.py

```diff
@@ -753,9 +778,11 @@ def to_value(self, unit=None, equivalencies=[]):
                 if not is_effectively_unity(scale):
                     # not in-place!
                     value = value * scale
+        else:
+            value = self._to_value(unit, equivalencies)
 
         # Index with empty tuple to decay array scalars in to numpy scalars.
```

Comment:
Needs an expanded comment.

Reply:
Done

Reply:
Actually, not sure I answered the right comment... The line here ensures that like for `ndarray`, if `Quantity` is a scalar, its value will be a numpy scalar rather than an array scalar. I must admit that I've never been sure this was really the right approach, but out of scope here...  The line itself just means we do not take yet another view for the case that it is not a scalar. Admittedly is out-of-scope, so can remove if need be.